.github/workflows/abi.yaml

```diff
@@ -88,19 +90,15 @@ jobs:
       run: |
         BUILDER_IMAGE="postgres:${{matrix.builder}}-alpine"
 
-        # Recent versions of alpine have OpenSSL 3 as the
-        # default version which is unfortunately not packaged
-        # for the earliest versions we check against in this
-        # test. So we pin the version we compile against to
-        # OpenSSL 1.1 in the backwards test.
-        if [[ "${{ matrix.test }}" == *backward ]]; then
-          EXTRA_PKGS="openssl1.1-compat-dev"
-        else
-          EXTRA_PKGS="openssl-dev"
-        fi
-
+        docker pull ${BUILDER_IMAGE}
+        docker buildx imagetools inspect ${BUILDER_IMAGE}
         docker run -i --rm -v $(pwd):/mnt -e EXTRA_PKGS="${EXTRA_PKGS}" ${BUILDER_IMAGE} bash <<"EOF"
           apk add cmake gcc make build-base krb5-dev git ${EXTRA_PKGS}
+          # We run the same extension on different docker images, old versions
+          # have OpenSSL 1.1 and the new versions have OpenSSL 3, so we try to
+          # pin the 1.1. Note that depending on PG version, both images might
+          # have 1.1 or 3, so we just try to install both these packages.
```

Comment:
I think this comment is misleading. In the line below, we install `openssl-dev`  only if `apk add openssl1.1-compat-dev` returns an error.

Reply:
I reworded it.